dashboards/vmagent.json

```diff
@@ -1,5 +1,14 @@
 {
```

Comment:
A general recommendation to all panels in Grafana - use soft min: 0 for Y ordinate. This helps to avoid false impression of series changes depending on the time interval.